### PR TITLE
GitHub workflow: Start progress

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**
Assign to sender when card is moved from `To do` to `In progress` column on the Best Kanban

### CODE REVIEW

File: .github/workflows/StartProgress.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+on:
+  project_card:
+    types: ["moved"]
+
+jobs:
+  assign_to_self:
+    runs-on: ubuntu-latest
+    if: |
+        github.event.changes.column_id.from == 4971951
```
Comment (andrei-epure-sonarsource) [inline .github/workflows/StartProgress.yml:11]:
```suggestion
        github.event.changes.column_id.from == 4971951 # To Do
```

File: .github/workflows/StartProgress.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        && github.event.project_card.column_id == 4971952
```
Comment (andrei-epure-sonarsource) [inline .github/workflows/StartProgress.yml:12]:
```suggestion
        && github.event.project_card.column_id == 4971952 # In Progress
```

File: .github/workflows/StartProgress.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            //
+            const url = context.payload.project_card.content_url;
+            console.log("Processing " + url);
+            var issue = (await github.request(url)).data;
+            if(!issue.assignee){
+                addAssignee(issue, context.payload.sender.login);
+            }
+            console.log(`Done`);
+
+
```
Comment (andrei-epure-sonarsource) [inline .github/workflows/StartProgress.yml:39]:
```suggestion

```

**Review by andrei-epure-sonarsource — APPROVED** [review]
LGTM (I left some suggestions before merging)

File: .github/workflows/StartProgress.yml (hunk 1 above)
Comment (pavel-mikula-sonarsource) [reply .github/workflows/StartProgress.yml:11]: It's not possible to add comments inside `if`. It doesn't work as `//` nor as `#`. See this run:

https://github.com/pavel-mikula-sonarsource/GitHubActionPlayground/actions/runs/400781201

File: .github/workflows/StartProgress.yml (hunk 2 above)
Comment (pavel-mikula-sonarsource) [reply .github/workflows/StartProgress.yml:12]: Same here

File: .github/workflows/StartProgress.yml (hunk 1 above)
Comment (pavel-mikula-sonarsource) [reply .github/workflows/StartProgress.yml:11]: For reference, run with `//` is one of the old/early versions: https://github.com/pavel-mikula-sonarsource/GitHubActionPlayground/actions/runs/293418742

**Pull request merged**